instrumentation/akka-http-10.0/src/main/java/io/opentelemetry/javaagent/instrumentation/akkahttp/AkkaHttpClientTracer.java

```diff
@@ -68,6 +68,6 @@ protected String responseHeader(HttpResponse httpResponse, String name) {
 
   @Override
   protected String getInstrumentationName() {
-    return "io.opentelemetry.auto.akka-http";
+    return "io.opentelemetry.javaagent.akka-http";
```

Comment:
Should we drop javaagent from instrumentation names? Don't think we're using it in library-based agent instrumentation.

Reply:
see #1436